packages/front-end/tsconfig.spec.json

```
@@ -16,7 +16,7 @@
     "src/**/*.test.jsx",
     "src/**/*.spec.jsx",
     "src/**/*.d.ts"
-  ],
+, "src/app/familytree.d.tsx"  ],
```

```suggestion
, "src/**/*.d.tsx"  ],
```
should give the same result but more consistency